.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: node_js
 sudo: false
 node_js:
-  - "4.4.3"
+  - "6.11.3"
 before_install:
-  - npm install -g npm@2.13.5
-  - npm install -g grunt-cli
+  - npm install -g npm@3
```

Comment:
We probably don't actually need this before_install block at all, now that I think about it...